packages/myst-to-typst/src/container.ts

```diff
@@ -41,6 +41,7 @@ export const containerHandler: Handler = (node, state) => {
     });
     return;
   }
+  state.useMacro('#show figure: set block(breakable: true)');
```

Comment:
Should this be in the theme? Can you explain what this is doing?

Reply:
This allows figures to span multiple pages - https://github.com/typst/typst/pull/1121 - without it tables stop at the bottom of the page and all the remaining rows are mushed together.

I think moving it to theme is probably a good idea (I don't know why we would ever _not_ want this), but I'm still playing with table formatting some (`breakable` does not solve width issues...) so this may not be the final solution...